### PR TITLE
feat(sdk-coin-injective): injective sdk init along with testcases

### PR DESCRIPTION
**Pull request by DinshawKothari**
Ticket: BG-79064

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -868,21 +868,21 @@ export const coins = CoinMap.fromCoins([
     '5f9506c5-f10a-43c2-92d3-52941083bbc7',
     'injective',
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:869]: Not sure why this was changed. Our pattern would be INJ and inj. Same with Networks.main.inj

Comment (mmcshinsky-bitgo) [reply modules/statics/src/coins.ts:869]: I would recommend keeping to established patterns and fixing these.

File: modules/account-lib/package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     "@bitgo/sdk-coin-eth2": "^1.2.27",
     "@bitgo/sdk-coin-hash": "^1.1.0",
     "@bitgo/sdk-coin-hbar": "^1.3.27",
+    "@bitgo/sdk-coin-injective": "^1.1.2",
```
Comment (mmcshinsky-bitgo) [inline modules/account-lib/package.json:44]: Is this required in account-lib for other services or can we remove it?

Comment (DinshawKothari) [reply modules/account-lib/package.json:44]: Yes this will be required for Wallet Platform.

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (DinshawKothari) [reply modules/statics/src/coins.ts:869]: We have inj coin name already used for erc20 inj token. That is why we have used injective for this.

Comment (mmcshinsky-bitgo) [reply modules/statics/src/coins.ts:869]: Gotcha. Another name collision :(

File: modules/sdk-coin-injective/src/tinjective.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Testnet Tia
```
Comment (Vijay-Jagannathan) [inline modules/sdk-coin-injective/src/tinjective.ts:2]: nit: typo; should be injective

**Review by Vijay-Jagannathan — APPROVED** [review]

**Pull request merged**